specificatie/Waardelijsten/openapi.yaml

```diff
@@ -222,37 +223,71 @@ components:
       required: false
       schema:
         type: string
+        example: "AardZakelijkRecht"
     waardelijstidentificatie_path:
       name: waardelijstidentificatie
       in: path
       description: "De identificatie, behorende bij de waardelijst."
       required: true
       schema:
         type: string
+        example: "AardZakelijkRecht"
     code:
       name: code
       in: query
       description: "De code, behorende bij de waardelijstwaarde."
       required: false
       schema:
         type: string
+        example: "6"
```

Comment:
Wat is het verschil tussen 'code' en 'code_path'?

Reply:
code is een query-parameter, code_path is een path-parameter